Packs/DuoAdminApi/Integrations/DuoAdminApi/DuoAdminApi.py

```diff
@@ -330,6 +338,63 @@ def delete_u2f_token(token_id):
     demisto.results('Token with ID ' + token_id + ' deleted successfully')
 
 
+def get_all_bypass_codes():
+    res = admin_api.get_bypass_codes()
+    entry = get_entry_for_object(
+        'Bypass', res, res,
+        {
+            'DuoAdmin.UserDetails(val.bypass_code_id==obj.bypasscodeid)': res
+        },
+        headers=[
+            'bypass_code_id',
+            'admin_email',
+            'expiration',
+            'reuse_count',
+            'user.created',
+            'user.email',
+            'user.last_login',
+            'user.status',
+            'user.user_id',
+            'user.username'
+        ]
+    )
+    demisto.results(entry)
+
+
+def get_all_admins():
+    res = admin_api.get_admins()
+
+    entry = get_entry_for_object(
+        'Admins', res, res,
+        {
+            'DuoAdmin.UserDetails(val.name==obj.name)': res
+        },
+        headers=[
+            'admin_id',
+            'admin_units',
+            'created',
+            'email',
+            'last_login',
+            'name',
+            'phone',
+            'role',
+            'status'
+        ]
+    )
+    demisto.results(entry)
+
+
+def modify_admin_user(user_id, status):
+    admin_api.update_admin(user_id, name, phone, password, password_change_required, status)
+    demisto.results('Status for' + user_id + ' Successful updated to ' + status)
+
+
+def modify_user(user_id, status):
+    admin_api.update_user(user_id, username, realname, status, notes, email,
+                          firstname, lastname, alias1, alias2, alias3, alias4, aliases)
```

Comment:
I dont understand the comment.  Can you please elaborate?

Reply:
Yes, the modify_user method gets as input only two variables (user_id, status).
The other variables such as username, realname,..., etc need to be defined in the method.
If you will try to open this on IDE you can see that an error is been thrown because these variables are unknown.
<img width="876" alt="image" src="https://user-images.githubusercontent.com/62752352/181456208-dc9538fd-6de7-4cd7-a164-874b8c82b9c5.png">